src/robust.jl

```diff
@@ -0,0 +1,110 @@
+# Robust Statistics
+
+#############################
+#
+#   Trimming outliers
+#
+#############################
+
+# Trimmed set
+"""
+    trim(x, p=0.2)
```

Comment:
What's the motivation for a default of 0.02? If it were me I'd omit a default and require the user to always specify the proportion to remove from the tails.